src/components/SignupForm/useFormFieldsConfig.tsx

```diff
@@ -0,0 +1,55 @@
+import SignUpFormDataInterface from './interfaces/SignupFormDataInterface';
+import FormFieldConfigInterface from './interfaces/FormFieldConfigInterface';
+
+const useFormFieldsConfig = (
+  formData: SignUpFormDataInterface,
+  errorMessageVisible: {
+    name: boolean;
+    email: boolean;
+    password: boolean;
+    repeatPassword: boolean;
+  }
+): FormFieldConfigInterface[] => {
+  const formFieldsConfig = [
+    {
+      key: 'input1',
```

Comment:
I don't like keys like input1, input2... you don't need keys jere. You could use labels for it